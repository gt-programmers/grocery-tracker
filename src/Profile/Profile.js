import React from 'react';
import './Profile.scss';

const Profile = () => {
  return (
    <section className="profile">
      <p>Profile</p>
    </section>
  )
}

export default Profile;