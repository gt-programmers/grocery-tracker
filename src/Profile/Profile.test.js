import React from 'react';
import Profile from './Profile';
import { shallow } from 'enzyme';

describe("Profile", () => {
  let wrapper = shallow( <Profile />)


  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})