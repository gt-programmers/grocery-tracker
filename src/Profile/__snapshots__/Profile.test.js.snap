// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should match the snapshot 1`] = `
<section
  className="profile page"
>
  <p>
    Profile
  </p>
</section>
`;
